config/cscs.py

```diff
@@ -924,12 +943,48 @@
             ]
         },
         {
-            'name': 'PrgEnv-aocc',
+            'name': 'PrgEnv-intel',
+            'target_systems': [
+                'pilatus'
+            ],
+            'modules': [
+                {'name': 'PrgEnv-intel', 'collection': True}
+            ]
+        },
+        {
+            'name': 'cpeAMD',
             'target_systems': [
                 'eiger', 'pilatus'
             ],
             'modules': [
-                {'name': 'PrgEnv-aocc', 'collection': True}
+                {'name': 'cpeAMD', 'collection': False}
```

Comment:
When a module is not a collection, just use the classical way of defining it: `'modules': ['cpeAMD']`.

Reply:
I did that, but I got an error so I changed it... maybe the error was due to something else, I can revert the change.

Reply:
I realised that initially I had specified `{'name': 'cpeAMD'}` without the `collection`, therefore I got the error.